apps/admin/reply-urls-operator/demo/deployment-patch.yaml

```diff
@@ -9,4 +9,4 @@ spec:
     spec:
       containers:
       - name: manager
-        image: sdshmctspublic.azurecr.io/reply-urls-operator:prod-3ee7540-20220920152807
+        image: sdshmctspublic.azurecr.io/reply-urls-operator:prod-2ab2c21-20230112091017
```

Comment:
is there a reason it's not using image automation?

Reply:
at the moment its just because of the crd changing etc